eventkit_cloud/core/urls.py

```diff
@@ -25,5 +25,7 @@
     url(r'^api/docs$', schema_view),
     url(r'^api/', include(router.urls, namespace='api')),
     url(r'^api/', include('rest_framework.urls', namespace='rest_framework')),
+#    url('^inbox/notifications/', include(notifications.urls, namespace='notifications')),
```

Comment:
Is there a reason this is left in here commented out?
Also, if you hit `/api` the list of endpoints does not have notifications as a option. Could you add that so its not hidden away?
![no_api_url_listed](https://user-images.githubusercontent.com/16799449/39721121-688a2342-520c-11e8-90a3-4f772b109f9a.PNG)

Reply:
Hmm, I'm not sure about this as @david-sherman was working on the notifications backend. I'm guessing that commented code was left in accidentally. And I think he added docs for swagger, but must have skipped the regular api docs.

David, do you want to take a look at this one?